openforcefield/typing/engines/smirnoff/plugins.py

```diff
@@ -0,0 +1,91 @@
+"""This module defines functions for loading parameter handler and parser clases which
+have been registered through the `entrypoint plugin system <https://packaging.python.
+org/guides/creating-and-discovering-plugins/#using-package-metadata>`_.
+
+.. warning ::
+
+   This feature is experimental and may be removed / altered in future versions.
+
+Currently only ``ParameterHandler`` classes can be registered via the plugin
+system. This is possible by registering the handler class through an entry
+point in your projects ``setup.py`` file::
+
+    setup(
+        ...
+        entry_points={
+            'openff.toolkit.plugins.handlers': ['CustomHandler = myapp:CustomHandler']
+        },
+        ...
+    )
```

Comment:
Just for my understanding, this approach is compatible with the current namespace, but it will just be taking up real estate in an otherwise empty namespace?

Reply:
I think that's correct.